OWNERS_ALIASES

```diff
@@ -32,6 +32,8 @@ aliases:
     - justinsb
     - luxas
     - timothysc
+    - fabriziopandini
```

Comment:
Alpha-sort please

Reply:
updated.

Reply:
actually i've missed the comment above about the sync with k/community. will update again.

Reply:
the list now looks like:

```
sig-cluster-lifecycle-leads:
    - fabriziopandini
    - justinsb
    - neolit123
    - timothysc
```

which matches k/community but is alpha-sorted.

Reply:
@neolit123 -- Sorry, I meant since you're already in the file, can you update the entire thing?

Reply:
@justaugustus if you want me to copy the OWNERS_ALIASES from k/community to k/enhancements minus "CUSTOM CONTENT" i can do that, but a minor side-effect is that the OWNERS_ALIASES in k/enhancements will no longer be alpha-sorted.

Reply:
Ahhh, well nvm. I'll need to fix up that list later. 
I don't want to hold this up on that.

Reply:
i think i can spin a quick tool in go that can be executed as part of CI in k/community.
WDYT?

Reply:
It'd be great to have something like that!

Reply:
ok, will send a PR.